Styles/main.css

```diff
@@ -77,21 +80,22 @@ header ul li{
     padding: 0;
     list-style-type: none;
     margin: 0;
+}
+header ul li a {

```

Comment:
You have a lot of declarations in the `header ul li` rule that should be applied to `header ul li a` instead